lib/commands/execute.js

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash';
+import { errors } from 'appium-base-driver';
+import log from '../logger';
+
+const commands = {};
+
+const DESKTOP_COMMANDS_MAPPING = {
+  setValue: 'desktopSetValue',
```

Comment:
`macos:commandNames` or `mac:commandNames` like this driver name instead of `desktop:commandName`

Reply:
changed